### PR TITLE
DJAN-107 tequila adoption

### PR DESCRIPTION
**Pull request by jbradberry**
Make the changes needed to conform to the Tequila adoption plan.
- [ ] move the roles out to their own separate repos
- [ ] rename and expand the functionality of the command-line interface

### CODE REVIEW

**Comment by dpoirier** [comment]
Can we update the install instructions in README?

**Comment by jbradberry** [comment]
Yep, I have that in progress.

File: tequila/tequila.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import argparse
+import os
+from subprocess import check_call
+
+import tequila
```
Comment (dpoirier) [inline tequila/tequila.py]: Don't need this anymore.

File: tequila/tequila.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+# TODO: Provide a way to pass additional arguments to ansible-playbook
+
+
+def touch(filename, content):
```
Comment (dpoirier) [inline tequila/tequila.py]: Don't need this anymore.

File: README.rst [hunk 3]
```diff
@@ -24,187 +31,207 @@ check out `tequila on Github <https://github.com/caktus/tequila>`_.
...
+  be placed in
+  ``$(PWD)/deployment/environments/<envname>/all/secrets.yml``.  All
+  variables defined in such encrypted files should then be exposed in
+  a corresponding non-encrypted file ::
 
-* Variables whose values should be secret should be declared in the same
-  files as other variables, depending on their scope, but their value
-  should be set to ``{{ secret_<varname> }}``::
+    ---
+    # file: deployment/group_vars/staging/secrets.yml
```
Comment (dpoirier) [inline README.rst]: this path doesn't match the one in the paragraph above (`deployment/environments/<envname>/all/secrets.yml`)

**Comment by dpoirier** [comment]
Instructions say to create `deployment/playbooks/site.yml` but tequila seems to be looking for `deployment/site.yml`.

**Comment by jbradberry** [comment]
Ah, right, I had to change that to make the pathing to the global vars files work, but forgot to change it here.

**Comment by dpoirier** [comment]
Can we change the example variable in the README to "db_password" instead of "DB_PASSWORD" since tequila-postgresql wants "db_password"?

**Comment by dpoirier** [comment]
Document how/where to set `env_name`, or else figure it out ourselves.

**Comment by dpoirier** [comment]
To work around lack of git branch support in galaxy until 2.1.0, let's suggest this syntax:

```
- src: https://github.com/caktus/tequila-common/archive/BRANCHNAME.tar.gz
  name: tequila-common
```

**Comment by jbradberry** [comment]
Closing as no longer relevant.

**Pull request closed without merging**